api/ipc.js

```diff
@@ -1299,10 +1289,9 @@ export async function request (command, params, options) {
 
   await ready()
 
-  const request = new globalThis.XMLHttpRequest()
   const signal = options?.signal
-  const index = globalThis?.__args?.index ?? 0
-  const seq = nextSeq++
+  const request = new globalThis.XMLHttpRequest()
+  const params = new IPCSearchParams(value, Date.now())
```

Comment:
I think that we can set `noonce` in constructor

Reply:
`send` doesn't set nonce